metrics/compose_linux.yaml

```diff
@@ -11,6 +11,35 @@ services:
     volumes:
       - ./prometheus:/etc/prometheus
       - prom_data:/prometheus
+
+  loki:
+    network_mode: "host" # -- only linux
+    image: grafana/loki:3.0.0
```

Comment:
Do we really need these network_mode: host things? Seems like I get everything connecting to each other without it. If we can drop that, we can clean up to have just one compose.yml